localisation/00_wc_government.csv

```diff
@@ -19,3 +19,6 @@ dragon_government;Dragonflight;Vol draconique;Drachenschwarm;;;;;;;;;;;x
 dragon_government_desc;The different dragon races are divided into dragonflights. Each flight was initially led by a Dragon Aspect, which were chosen by the Titans in the first place. For a long time, the most important large flights were the Blue, Black, Bronze, Green and Red Dragonflights. There were other, smaller flights, that were considered artificial or unnatural, such as the Infinite or the Chromatic Dragonflight, but no-one knows, what the future holds. In addition to the established dragonflights, there are also a few renegade and therefore independent dragons.;Les diff�rentes races de dragons sont divis�es en vols de dragons. Chaque vol �tait initialement dirig� par un Dragon Aspect, choisi par les Titans en premier lieu. Pendant longtemps, les grands vols les plus importants furent les vols de dragon bleu, noir, bronze, vert et rouge. Il y avait d'autres vols, plus petits, qui �taient consid�r�s comme artificiels ou artificiels, comme l'Infini ou le Vol Dragon Chromatique, mais personne ne sait ce que l'avenir nous r�serve. En plus des vols de dragons �tablis, il y a aussi quelques dragons ren�gats et donc ind�pendants.;Die verschiedenen Drachenrassen sind in Drachenschw�rme unterteilt. Jeder Schwarm wurde zun�chst von einem Drachenaspekt geleitet, der von den Titanen urspr�nglich ausgew�hlt wurde. Lange Zeit waren die wichtigsten gro�en Schw�rme die blauen, schwarzen, bronzenen, gr�nen und roten Drachenschw�rme. Es gab andere, kleinere Fl�ge, die als k�nstlich oder unnat�rlich galten, wie den unendlichen oder den chromatischen Drachenschwarm, aber niemand wei�, was die Zukunft bringt. Neben den etablierten Drachenschw�rmen gibt es auch einige abtr�nnige und damit unabh�ngige Drachen.;;;;;;;;;;;x
 society_government;Society;Soci�t�;Organisation;;;;;;;;;;;x
 society_government_desc;This society of individuals has progressed far beyond having common interests and motivations, and the membership of this society has grown to encompass entire populations of individuals. As such, this society has taken on more governmental reforms and leadership and is now recognized as a full, sovereign state contesting for resources like any other nation on Azeroth.;Les soci�t�s repr�sentent la forme politique des organisations.;Eine Regierungsform, die Gesellschaften und Organisationen widerspiegelt.;;;;;;;;;;;x
+hive_government;Hive;;;;;;;;;;;;;x
+hive_government_desc;An insectoid society built around kings and queens producing new members of the hive. Most larvae grow as submissive workers or soldiers, and only a few are born fertile and are able to reach the highest positions.;;;;;;;;;;;;x
+hive_government_FEATURES;�Y*�! Gets levy size bonus and dynasty opinion malus per every born child\n�Y*�! Can sing to the swarm every 50 years getting morale, opinion bonuses and build cost, tax maluses;;;;;;;;;;;;;x
```

Comment:
See below for my comments on the 'per born child' part. The second part had a few problems, so I tried to clean it up a little.
```suggestion
hive_government_FEATURES;�Y*�! Gets levy size bonus and dynasty opinion malus per child\n�Y*�! Can sing to the swarm every 50 years gaining morale and opinion bonuses, at the expense of higher building costs and tax maluses;;;;;;;;;;;;;x
```